packages/cli/cli_internal.ts

```diff
@@ -168,7 +174,7 @@ program
       const toIndex = tryGetScriptIndex(options.to)
       await deployAndSaveProgress(config, networkId, fromIndex, toIndex)
     } catch (error) {
-      program.error(`Failed to deploy contracts, error: ${(error as Error).stack}`)
+      program.error(`✘ Failed to deploy contracts, error: ${buildErrorOutput(error, options.debug)}`)
```

Comment:
If debug is enabled via the config file, should we also show the debug info? It looks good to keep the two debug flag consistent.

Reply:
> If debug is enabled via the config file, should we also show the debug info? It looks good to keep the two debug flag consistent.

Yeah good point, updated